### PR TITLE
Tls enable router metrics

### PR DESCRIPTION
**Pull request by lulf**

### CODE REVIEW

File: systemtests/scripts/run_test_component.sh [hunk 1]
```diff
@@ -23,6 +23,8 @@ popd
 
 $SYSTEMTESTS/scripts/collect_logs.sh $OC_PATH $ARTIFACTS_DIR
 
+oc get pods
+
 if [ $failure -gt 0 ]
```
Comment (rgodfrey) [inline systemtests/scripts/run_test_component.sh]: why is this here?

Comment (lulf) [reply systemtests/scripts/run_test_component.sh]: Useful to see if things crash :) i'd like to keep it in
